scripts/releases/prebuild/compose-framework.js

```diff
@@ -0,0 +1,143 @@
+  execSync(command, {stdio: 'inherit'});
+
+  // Copy bundles into the framework
+  copyBundles(scheme, dependencies, output, frameworkPaths);
+}
+
+/**
+ * Copies the bundles in the source frameworks to the target xcframework inside the xcframework's Resources folder
+ */
+function copyBundles(
+  scheme /*: string */,
+  dependencies /*: $ReadOnlyArray<Dependency> */,
+  outputFolder /*:string*/,
+  frameworkPaths /*:Array<string>*/,
+) {
+  console.log('Copying bundles to the framework...');
+
+  // Let's precalculate the target folder. It is the xcframework's output folder with
+  // all its targets.
+  const targetArchFolders = fs
+    .readdirSync(outputFolder)
+    .map(p => path.join(outputFolder, p))
+    .filter(p => fs.statSync(p).isDirectory());
+
+  // For each framework (in frameworkPaths), copy the bundles from the source folder.
+  // A bundle is the name of the framework + _ + target name + .bundle. We can
+  // check if the target has a bundle by checking if it defines one or more resources.
+  frameworkPaths.forEach(frameworkPath => {
+    dependencies.forEach(dep => {
+      const resources = dep.files.resources;
+      if (!resources || resources.length === 0) {
+        return;
+      }
+
+      // Get bundle source folder
+      const bundleName = `${scheme}_${dep.name}.bundle`;
+      const sourceBundlePath = path.join(frameworkPath, '../../', bundleName);
+      if (fs.existsSync(sourceBundlePath)) {
+        // Target folder - needs to be copied to the resulting framework
+        targetArchFolders.forEach(targetArchFolder => {
+          const targetBundlePath = path.join(
+            targetArchFolder,
+            `${scheme}.framework`,
+            'Resources',
+            bundleName,
+          );
+          if (
+            !fs.existsSync(path.join(targetArchFolder, `${scheme}.framework`))
+          ) {
+            console.warn("Target Bundle path doesn't exist", targetBundlePath);
+          }
+          if (!fs.existsSync(path.dirname(sourceBundlePath))) {
+            console.warn("Source bundle doesn't exist", sourceBundlePath);
+          }
+
+          // A bundle is a directory, so we need to copy the whole directory
+          copyDirectory(sourceBundlePath, targetBundlePath);
+        });
+      } else {
+        console.warn(`Bundle ${sourceBundlePath} not found`);
+      }
+    });
+  });
+}
+
+/**
+ * Copies a directory recursively
+ */
+function copyDirectory(source /*: string*/, target /*: string*/) {
+  if (!fs.existsSync(target)) {
+    fs.mkdirSync(target, {recursive: true});
+  }
+  // Read all files in source
+  const files = fs.readdirSync(source);
+  files.forEach(file => {
+    const sourceFile = path.join(source, file);
+    const targetFile = path.join(target, file);
+    if (fs.statSync(sourceFile).isDirectory()) {
+      copyDirectory(sourceFile, targetFile);
+    } else {
+      fs.copyFileSync(sourceFile, targetFile);
+    }
+  });
+}
```

Comment:
I'm definitiely not a JS expert, but why implementing the copy like this, instead of using the `execSync(`mv -p src dest)` command?
This is one line that relies on a well known working code. While reimplementing the move might bring in some bugs.

Perhapst there are some benefits that I'm not aware of...

Reply:
Yeah, just me not thinking clearly. Fixed!